examples/step-49/doc/intro.dox

```diff
@@ -99,13 +99,337 @@ many cases it is best to combine both approaches. The file can be easily
+point by point but we will only be needing to use curves.  This assigns
+boundary IDs to each vertex on the boundary that we choose.  We will see
+in <code>step-49.cc</code> that this is useful for mesh refinements using
+deal.ii.
+
+We assign boundary ID 0 to outer boundary.  This is usually here by default
+but we include it anyway.
+
+@code
+Physical Curve(0) = {38, 39, 52, 53, 54, 55, 56, 57, 59, 60, 58, 40, 41, 4, 5, 6, 7, 8, 9};
+@endcode
+
+Assign boundary ID of 1 top right circle.
+
+@code
+Physical Curve(1) = {12};
+@endcode
+
+Assign boundary IDs of 2 & 3 to the top & bottom half of the bottom left
+hole respectively.  We can see here that we needed to pick two boundaries
+to make both of the physical curves.  Recall this circle is comprised of
+four circle arcs, so Physical Curve(2) is the upper half and Physical
+Curve(3) is the lower half of the lower right circle.
+
+@code
+Physical Curve(2) = {48, 49};
+Physical Curve(3) = {50, 51};
+@endcode
+
+Assigns Physical Surface to the curves
+
+@code
+Physical Surface(2) = {1, 2, 3, 4};
+@endcode
+
+<h5>Meshing Parameters</h5>
+
+This can all be adjusted for viewing on the GMSH UI.  We include it
+at the bottom of our .geo file to keep the changes when creating our
+<code>.msh</code> file.
+
+@code
+Mesh.Algorithm = 3;
+Mesh.RecombineAll = 1;
+Mesh.CharacteristicLengthFactor = .6;
+Mesh.SubdivisionAlgorithm = 1;
+Mesh.Smoothing = 20;
+Show "*";
+@endcode
+
+To view what each of these do, press <code>Tools -> Options -> Mesh</code>.
+
+You may change the algorithm used for meshing in <code>2D algorithm</code>.
+
+<code>RecombineAll</code> takes the triangular mesh created from whatever
+algorithm was used to mesh and combines them into rectangular meshes.  This
+is crucial because deal.ii will not read in triangular meshes.  In the
+Options this can be found as a checkbox, which should be checkmarked.
+
+<code>CharacteristicLengthFactor</code> can be thought of the distance between
+nodes on the boundaries of the mesh.  This sets the initial size of the nodes
+of the mesh.  If we want a finer mesh we want a smaller characteristic length.
+For this examples purposes we would like to do more refinement in deal.ii so
+we pick a relatively coarse mesh.  This can also be adjusted in the Options
+menu.
+
+<code>SubdivisionAlgorithm</code> is set to 1 in our file because we want to
+use "All Quads" for subdivision.  In the options menue the default is "None",
+but we use "All Quads" for reasons mentioned earlier.
+
+<code>Smoothing</code> is a post-processing step that iteratively deforms the
+vertices.  For our example we pick 20, however this value can be adjusted
+accordingly, as it's not the most important step in mesh building.
+
+
+
+@note That ends the code for <code>example.geo</hcode>.  It's important to note that
+  if we don't type in the <code>.geo</code> file like above, the .geo file will not
+  come in nearly as satisfying.  This code was made and then edited to look more
```

Comment:
I am not sure what you are trying to say in this paragraph. Maybe rephrase or delete?